### PR TITLE
Add Normalization splitk instances

### PR DESCRIPTION
**Pull request by rocking5566**
1. Add normalization instances
2. Fix splitK bug for some problem size

### CODE REVIEW

File: include/ck/tensor_operation/gpu/grid/normalization/gridwise_normalization_splitk_1st.hpp [hunk 1]
```diff
@@ -193,10 +193,13 @@ struct GridwiseNormalizationSplitK1st
         auto var_global_val_buf = make_dynamic_buffer<AddressSpaceEnum::Global>(
             p_variance_global, mean_var_grid_desc_m_kblock.GetElementSpaceSize());
 
-        auto threadwise_welford = ThreadwiseWelford();
-        int kRaw                = x_grid_desc_m_k.GetTransforms()[I2].GetUpperLengths()[I0];
-        threadwise_welford.max_count_ =
-            GetKPerThread(kRaw, k_grid_size, block_k_cluster_id, thread_k_cluster_id);
+        auto threadwise_welford       = ThreadwiseWelford();
+        int kRaw                      = x_grid_desc_m_k.GetTransforms()[I2].GetUpperLengths()[I0];
```
Comment (qianfengz) [inline include/ck/tensor_operation/gpu/grid/normalization/gridwise_normalization_splitk_1st.hpp:198]: Can you calculate `kRaw` from the original dimension lengths before merging ?

File: include/ck/tensor_operation/gpu/grid/normalization/gridwise_normalization_splitk_1st.hpp [hunk 2]
```diff
@@ -78,14 +78,14 @@ struct GridwiseNormalizationSplitK1st
...
+    GetKPerThread(int k, int kRaw, int kGridSize, int block_k_cluster_id, int thread_k_cluster_id)
     {
         bool is_rightmost_block = block_k_cluster_id == kGridSize - 1;
 
         if(is_rightmost_block)
         {
-            int left_kPerBlock = math::integer_divide_ceil(kRaw, kGridSize);
-            int kPerBlock      = kRaw % kGridSize == 0 ? left_kPerBlock : kRaw % left_kPerBlock;
+            int left_kPerBlock = math::integer_divide_ceil(k, kGridSize);
+            int kPerBlock      = kRaw - left_kPerBlock * (kGridSize - 1);
```
Comment (qianfengz) [inline include/ck/tensor_operation/gpu/grid/normalization/gridwise_normalization_splitk_1st.hpp]: Using better name to replace `left_kPerBlock`, `kPerBlock`  like `kSizePerBlock`,  `kSizeRightmostBlock`

Comment (rocking5566) [reply include/ck/tensor_operation/gpu/grid/normalization/gridwise_normalization_splitk_1st.hpp]: b41c98f411da01febf80a9094e3a441240123c0d

File: include/ck/tensor_operation/gpu/grid/normalization/gridwise_normalization_splitk_1st.hpp (hunk 1 above)
Comment (rocking5566) [reply include/ck/tensor_operation/gpu/grid/normalization/gridwise_normalization_splitk_1st.hpp:198]: I will refine the flow in next PR. This PR focus on adding the instance

**Review by qianfengz — DISMISSED** [review]

**Review by qianfengz — APPROVED** [review]

**Pull request merged**